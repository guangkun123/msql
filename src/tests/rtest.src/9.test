# Force a duplicate user and do a distinct ordered join

insert into test2 (user, fullname) values ('fred','Fred Smith') \p\g

select distinct test2.user, test.age
        from test2,test
        where test2.user = test.user
        order by test2.user
\p\g

