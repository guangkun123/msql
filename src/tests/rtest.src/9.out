
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     -> 
Query buffer
------------

insert into test2 (user, fullname) values ('fred','Fred Smith') 
[continue]
    -> 
Query OK.




mSQL >     ->     ->     ->     ->     -> 
Query buffer
------------
select distinct test2.user, test.age
        from test2,test
        where test2.user = test.user
        order by test2.user

[continue]
    -> 
Query OK.

3 rows matched.

 +------------+----------+
 | user       | age      |
 +------------+----------+
 | Bambi      | 26       |
 | fred       | 31       |
 | paulp      | 26       |
 +------------+----------+



mSQL >     -> 
Bye!

